lib/Zonemaster/CLI.pm

```diff
@@ -147,15 +147,13 @@ has 'restore' => (
 has 'ipv4' => (
     is      => 'ro',
     isa     => 'Bool',
-    default => 1,
```

Comment:
This default behavior is not moved into the code. I mean, before the change if you do not have the `net` stanza in the profile file and you just run the CLI, IPv4 and IPv6 will be enabled by default.

Here with the current changes, removing the stanza from the profile file gives a CRITICAL message :
```
   0.01 CRITICAL  Both IPv4 and IPv6 are disabled.
```

I feel an `else` statement is needed in the code to enable IPv4 and IPv6 by default.

Reply:
I had to remove that "default" to make it possible to detect if the option had been actively set or not.

Reply:
Now I see what you mean. If the default profile.json does not have any IPv4 or IPv6 enabled, then there will no network enabled. -- In Backend both are enabled if both are disabled.